src/resources/auth/auth.repository.js

```diff
@@ -0,0 +1,37 @@
+const bcrypt = require('bcrypt');
+const { users } = require('../../loader/dbconnect');
+// const HttpError = require('../../tools/httpError');
+
+function getUserByEmail(email) {
+  return users.find((c) => c.email === email);
+}
+
+function register(user) {
+  users.push({
+    id: Math.random(),
```

Comment:
for generate id will be better to use uuid. https://www.npmjs.com/package/uuid

Reply:
yeap